web/src/reactions.js

```diff
@@ -77,6 +79,11 @@ function update_ui_and_send_reaction_ajax(message_id, reaction_info) {
     const has_reacted = current_user_has_reacted_to_emoji(message, local_id);
     const operation = has_reacted ? "remove" : "add";
     const reaction = create_reaction(message_id, reaction_info);
+    const message_reaction_id = local_id + message_id;
```

Comment:
`local_id` is a string, of `reaction_type,emoji_code`, and `message_id` is an int -- for cleanliness, it seems like mayeb we should join these with a `,` or similar so it's easier to look at for debugging, and so we eliminate a really unlikely collision in `emoji_code` + `message_id` when the border between them isn't delimited in any way.

Reply:
I am agree with you! Fixed!